apps/docs/content/components/River/index.mdx

```diff
@@ -174,6 +177,12 @@ Always use high quality images and videos. Avoid using images with text or singl
 
 The image is automatically styled to fit the width of the parent container. If you have bespoke styling requirements, you can disable the provided default styles and style the image manually.
 
+### Additional guidance
+
+- We recommend using a river left as the first river of the flow to start with a visual and avoid stacking text with above section on mobile
```

Comment:
Nit. In RTL reading directions, this would be right-aligned, so could we use directionally-agnostic terms here if possible. E.g. start-aligned if you're referring to 'left'. This is going to become more important as over time, as we expand on i18n.

Could this guidance go into the stacking section too? Feels kind of relevant to that, unless I've misinterpreted it.

Reply:
I see what you mean by directionally agnostic. I was intending to only talk about the layout and not the text. I don't see any issue with this advice being applied in a RTL language context. Perhaps a different wording could solve this (will make a suggestion)